test/functional/plugin/man_spec.lua

```diff
@@ -0,0 +1,181 @@
+local helpers = require('test.functional.helpers')(after_each)
+local plugin_helpers = require('test.functional.plugin.helpers')
+
+local Screen = require('test.functional.ui.screen')
+
+local buffer, command, eval = helpers.buffer, helpers.command, helpers.eval
+
+before_each(function()
+  plugin_helpers.reset()
+  helpers.clear()
+  command('syntax on')
+  command('set filetype=man')
+end)
+
+describe('In autoload/man.vim', function()
+  describe('function man#highlight_formatted_text', function()
+    local screen
+
+    before_each(function()
+      command('syntax off') -- Ignore syntax groups
+      screen = Screen.new(52, 5)
+      screen:attach()
+    end)
+
+    after_each(function()
+      screen:detach()
+    end)
+
+    local function expect(string)
+      screen:expect(string,
+      {
+        b = { bold = true },
+        i = { italic = true },
+        u = { underline = true },
+        bi = { bold = true, italic = true },
+        biu = { bold = true, italic = true, underline = true },
+      },
+      {{ bold = true, foreground = Screen.colors.Blue }})
+    end
+
+    local function expect_without_highlights(string)
+      screen:expect(string, nil, true)
+    end
+
+    local function insert_lines(...)
```

Comment:
`helpers.insert`

Reply:
I added this because the text contains a lot of control characters, and I found
```lua
insert_lines("i\bi")
```
more readable than
```lua
helpers.insert("i<C-v><C-h>i")
```
Would you prefer the second form?

Reply:
@keidax The second form is preferred, it's the prevailing convention in our tests and throughout Vim-land.